cli/main.go

```diff
@@ -197,7 +197,7 @@ func main() {
 					Action: func(c *cli.Context) error {
 						ozoneClient := api.CreateOzoneClient(c.GlobalString("om"))
 						address := OzoneObjectAddressFromString(c.Args().Get(0))
-						f, err := os.Open("/tmp/asd")
+						f, err := os.Open(c.Args().Get(1)
```

Comment:
This change improves flexible IMHO.

Thanks @adoroszlai for the contribution, 
+1 (Let's merge it after importing POC code.)